HintList.py

```diff
@@ -524,6 +524,8 @@ def tokens_required_by_settings(world: World) -> int:
     'Ocarina C down Button':                                    (["a low note"], "the Ocarina C down Button", 'item'),
     'Ocarina C left Button':                                    (["a somewhat high note"], "the Ocarina C left Button", 'item'),
     'Ocarina C right Button':                                   (["a middle note"], "the Ocarina C right Button", 'item'),
+    'Fairy Drop':                                               (["an annoying companion"], "a Stray Fairy", 'item'),
```

Comment:
We should make sure the cryptic hints make sense between bottled and stray fairies. I think “Navi's cousin” sounds more like a stray fairy than a bottled one, so I suggest adding it here and removing it from the `Bottle with Fairy` hints.

```suggestion
    'Fairy Drop':                                               (["an annoying companion", "Navi's cousin"], "a Stray Fairy", 'item'),
```

Reply:
Remember to also remove this hint from the bottled fairy item.